{"name": "Проверка, что пользователь не может авторизоваться с неверным паролем", "status": "passed", "steps": [{"name": "Отправка POST-запроса на эндпоинт: 'auth/login'", "status": "passed", "parameters": [{"name": "endpoint", "value": "'auth/login'"}, {"name": "headers", "value": "None"}, {"name": "json", "value": "{'email': 'fltwaxkwit@test.ru', 'password': 'jneeugsahl2'}"}], "start": 1727672037262, "stop": 1727672037843}], "start": 1727672037262, "stop": 1727672037843, "uuid": "25a10d25-4015-4fd5-9b29-bb683af53587", "historyId": "bb369f30f0579ede7a63c84458d40383", "testCaseId": "bb369f30f0579ede7a63c84458d40383", "fullName": "tests.test_login.TestLogin#test_login_user_error_nonexistent_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-DPIGL2N"}, {"name": "thread", "value": "35344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}